packages/kit/src/views/Staking/components/StakingActivityIndicator/index.tsx

```
@@ -43,6 +45,92 @@
+      if (!isPending || !accountId || !stakeTag) {
+        return;
+      }
+
+      await backgroundApiProxy.serviceHistory.fetchAccountHistory({
+        accountId,
+        networkId,
+      });
+
+      await refreshPendingTxs();
+    },
+    [isPending, accountId, networkId, stakeTag, refreshPendingTxs, pollingInterval],
```

ESLint

[error] React Hook usePromiseResult has an unnecessary dependency: 'pollingInterval'. Either exclude it or remove the dependency array.

(react-hooks/exhaustive-deps)